### PR TITLE
Some performance tuning

### PR DESCRIPTION
**Pull request by TonioGela**
My hands wrote this PR by their own

### CODE REVIEW

File: src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/package.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package it.pagopa.interop.backendforfrontend.service
+
+import scala.concurrent.Future
+import it.pagopa.interop.commons.utils.extractHeaders
+
+package object impl {
+  def withHeaders[T](
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/package.scala:7]: this function may have the right to stay in commons

**Review by galales — APPROVED** [review]
😍

File: src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/package.scala (hunk 1 above)
Comment (TonioGela) [reply src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/package.scala:7]: it does in fact

**Review by beetlecrunch — APPROVED** [review]

**Comment by beetlecrunch** [comment]
🚀

**Pull request merged**